ts/private/ts_lib.bzl

```diff
@@ -22,7 +22,16 @@ STD_ATTRS = {
         doc = "https://www.typescriptlang.org/docs/handbook/compiler-options.html",
     ),
     "data": attr.label_list(
-        doc = "Runtime dependencies to include in binaries/tests that depend on this TS code",
+        doc = """Runtime dependencies to include in binaries/tests that depend on this library.
```

Comment:
Is this essentially a copy+paste of the `data` of other rules_js rules? Is there a common place in rules_js that could be linked to for a full explanation of the providers here?

Anyone without more understanding of rules_js won't understand anything beyond the first sentence here...

Reply:
Good point. We could expose the docstring in js_library_helpers... or a default data attribute that you can override with dicts add.

Reply:
Can we just have this first sentence followed by "See: <url to full explanation>"? Except that URL probably has to be version specific?

Reply:
I ended up exposing the attr.label_list() from js_library for downstream rules to use.